src/selectorNameBase.ts

```diff
@@ -69,7 +69,7 @@ class SelectorNameValidatorWalker extends Lint.RuleWalker {
   }
 
   private validateSelector(className: string, arg: ts.Node) {
-    if (arg.kind === SyntaxKind.current().ObjectLiteralExpression) {
+    if (arg && arg.kind === SyntaxKind.current().ObjectLiteralExpression) {
```

Comment:
Yes, but I cannot react fast, without seeing code snippets and you can not provide me some, which gives me only this option for now.

 I will invest more time, solving the issue, during the weekend.